k8s/helm/values.yaml

```diff
@@ -63,3 +63,5 @@ addressLookupApi:
 
 confirmationEmailLimit: 10
 feedbackLimit: 10
+
+eqSubmissionConfirmationBackend: "cloud_tasks"
```

Comment:
I don't think we prefix properties with `eq` (even though it's in the environment variable name).